src/main/scala/edu/twitter/model/impl/neuralnetwork/NeuralNetworkBuilder.scala

```diff
@@ -46,12 +44,11 @@ class NeuralNetworkBuilder(sc: SparkContext) extends GenericModelBuilder {
     }
 
     val batchSize = 256 //Number of examples in each minibatch
-    val nEpochs = 10 //Number of epochs (full passes of training data) to train on
+    val nEpochs = appConfig.neuralNetworkEpochs //Number of epochs (full passes of training data) to train on
     val truncateReviewsToLength = 280 //Truncate reviews with length (# words) greater than this
 
     //DataSetIterators for training and testing respectively
-    val dataPath = this.getClass.getClassLoader.getResource("labeled-tweets").getPath
-    val data = new TweetsLoader(sc).loadDataSet(dataPath)
+    val data = new TweetsLoader(sc).loadDataSet(appConfig.paths.trainingDataPath)
     val Array(trainData, testData) = data.randomSplit(Array(0.85, 0.15))
```

Comment:
Kindly check if the ratios are different in dev and prod.

Reply:
I think we need the same ratio in prod and dev.